package.json

```
@@ -1,6 +1,8 @@
 {
   "private": true,
   "scripts": {
+    "bad-linebreaks": "node ./scripts/bad-linebreaks.mjs",
+    "bad-linebreaks:fix": "node ./scripts/bad-linebreaks.mjs fix",
```

this isn't needed, since you can `npm run bad-linebreaks fix`

---

> this isn't needed, since you can `npm run bad-linebreaks fix`

This is self documenting the availability and usage of the `fix` option. Without it a contributor will have to look at bad-linebreaks.mjs script.

---

if it was turned into `--fix` it'd match eslint and there'd be no need to self-document it.

---

it's not needed, but strong preference for leaving it.  this is the place where people will look for this when CI fails, and `lint:fix` is a common pattern

---

```suggestion
    "bad-linebreaks:fix": "npm run bad-linebreaks fix",
```
then let's at least do this so we're not repeating the command

---

sure? 🤷